src/component/archive/archive-list-container.js

```diff
@@ -1,15 +1,80 @@
 import { connect } from 'react-redux';
-import ListComponent from './archive-list-component';
+import ArchiveList from './archive-list-component';
 import { fetchArchive, revive } from './../../store/archive-actions';
+import { updateSettingForGroup } from './../../store/settings/actions';
 
 const mapStateToProps = state => {
-    const archive = state.archive.get('list').toArray();
+    const featureMetrics = state.featureMetrics.toJS();
```

Comment:
this mostly looks like a copy of `feature/list-container.jsx`. Could we find a way to share that code?